doc/user/data/disaster_recovery/cluster.yml

```diff
@@ -0,0 +1,65 @@
+columns:
+  - column: "Failure Type"
+  - column: "RPO"
+  - column: "RTO (RF1 - single AZ)"
+  - column: "RTO (RF2 - multiple AZs)"
+
+rows:
+  - Failure Type: "**Machine failure**"
+    RPO: 0
+    RTO (RF1 - single AZ): |
+      Time to spin up new machine + possible rehydration time, depending on the
+      objects on the machine:
+
+      - If sources, no additional time (i.e., does not require rehydration).
+      - If sinks, no additional time (i.e., does not require rehydration).
+      - If compute, rehydration time.
+      - If serving, rehydration time.
+
+      To reduce rehydration time, scale up the cluster.
+    RTO (RF2 - multiple AZs): |
+      Can be:
+
+      - 0 if only compute and serving objects are on the machine.
+
+      - Time to spin up new machine if sources or sinks are on the machine.
```

Comment:
For kafka sources, I believe this isn't true anymore ... but, for v1, am not calling out kafka or webhook sources specifically, but lumping them.